torch/csrc/jit/tracer_state.h

```diff
@@ -55,6 +55,9 @@ struct TracingState : public std::enable_shared_from_this<TracingState> {
   std::mutex mutex;
   variable_list inputs; // Used only for the duration of first stage
 
+  bool creates_handles; // should python ops ever get handles? Or should
```

Comment:
Why would this ever be `true`? When do we need it?

Reply:
`torch.onnx.export` still uses the old tracing pathway where and we need do a bunch of yak shaving to remove it.

Reply:
What do we need `creates_handles` for now? Looks like it's no longer necessary for ONNX, because it will correctly handle traces that had it set to `false`.

Reply:
We need it because I am pretty sure the ONNX symbolics expect those handles to be there, and I want to keep that fix, which will take a lot of iteration with the onnx tests, as a separate pr.

Reply:
I thought ONNX never supported handles at all, because it never supported backward export, and that's what the handles were originally added for. There is a bunch of special cases in the ONNX pass to make sure we handle handles correctly, but they're mostly checking if the handle is actually unused (which it should be if no backwards occurred.)

In any case if you guys need this to unblock I wouldn't block on this.

Reply:
(But it would be nice to have a comment saying this is temporary, and solely for supporting ONNX)